kolibri/core/assets/src/vue/nav-bar/index.vue

```diff
@@ -69,20 +95,69 @@
           return values(TopLevelPageNames).includes(value);
         },
       },
+      navShown: {
+        type: Boolean,
+        required: true,
+      },
+      headerHeight: {
+        type: Number,
+        required: true,
+      },
+      width: {
+        type: Number,
+        required: true,
+      },
+    },
+    methods: {
+      navigate(option) {
+        if (option.href) {
+          window.location.href = option.href;
+        } else if (option.action) {
+          option.action();
+        }
+      },
+      toggleNav() {
+        if (this.mobile | this.tablet) {
+          this.$emit('toggleSideNav');
+        }
+      },
     },
-    data: () => ({
-      version: __version, // eslint-disable-line no-undef
-    }),
     computed: {
+      closeStyle() {
+        return {
+          fontSize: headerHeight/2 + 'px',
+          marginLeft: this.width/20 + 'px',
+          marginRight: this.width/20 + 'px',
```

Comment:
computation results in [non-standard margins](https://material.io/guidelines/layout/metrics-keylines.html)